ignore/src/types.rs

```diff
@@ -301,7 +301,8 @@ const DEFAULT_TYPES: &'static [(&'static str, &'static [&'static str])] = &[
     ("vimscript", &["*.vim"]),
     ("wiki", &["*.mediawiki", "*.wiki"]),
     ("webidl", &["*.idl", "*.webidl", "*.widl"]),
-    ("xml", &["*.xml", "*.xml.dist"]),
+    ("xml", &["*.xml", "*.xml.dist", "*.dtd", "*.xsl", "*.xslt", "*.xsd",
+        "*.xjb", "*.rng", "*.sch"]),
```

Comment:
Sorry to be a pain about this, but please format the code in a style that is consistent with the surrounding code. Please consult other file types, such as `systemd`, for how to format the list when it gets to be too long.

Reply:
No problem. Didn't notice the difference.

It did this change now.

(BTW: The definition of `asp` is 87 characters long. ;-))

Reply:
Yes, I don't always catch violations of the line lengths, especially if they're close. (Because the GitHub UI doesn't make it easy to see.) And I rarely edit this file myself any more, so I didn't even know about it. :-)